frontend/src/components/applyForm/widgets/budget/Budget424aSectionB.tsx

```diff
@@ -306,7 +305,7 @@ function Budget424aSectionB<
               </th>
 
               {/* Four activity columns */}
-              {COLUMNS.map((columnIndex) => {
+              {COLUMNS.slice(0, 4).map((columnIndex: number) => {
```

Comment:
seems like calls to `COLUMNS.slice(0, 4)` should be replaced with `activityColumnIndices` in order to make sure the number of columns stays consistent